presto-hive/src/main/java/com/facebook/presto/hive/HiveCompressionCodec.java

```diff
@@ -32,12 +32,12 @@ public enum HiveCompressionCodec
     NONE(null, CompressionKind.NONE, CompressionCodecName.UNCOMPRESSED, f -> true),
     SNAPPY(SnappyCodec.class, CompressionKind.SNAPPY, CompressionCodecName.SNAPPY, f -> true),
     GZIP(GzipCodec.class, CompressionKind.ZLIB, CompressionCodecName.GZIP, f -> true),
-    LZ4(null, CompressionKind.NONE, null, f -> f == PAGEFILE),
-    ZSTD(null, CompressionKind.ZSTD, null, f -> f == ORC || f == DWRF || f == PAGEFILE);
+    LZ4(null, CompressionKind.NONE, CompressionCodecName.LZ4, f -> f == PAGEFILE),
```

Comment:
First arg codec is not reuquired?

Reply:
Technically it is not always used, but I updated the PR to add them in the first arg as well.

Reply:
Actually, I realized this caused other test failures. There is no ZSTD codec from the hadoop libs, but there is for Lz4. I added the LZ4 one and kept ZSTD null